core/common/src/main/java/alluxio/conf/PropertyKey.java

```diff
@@ -586,6 +586,16 @@ public String toString() {
           .setScope(Scope.SERVER)
           .setIsHidden(true)
           .build();
+  public static final PropertyKey GRPC_REFLECTION_ENABLED =
+      booleanBuilder(Name.GRPC_REFLECTION_ENABLED)
+          .setDefaultValue(false)
+          .setDescription("If true, grpc reflection will be enabled on master so that "
```

Comment:
if it only works on master then MASTER_GRPC_DEBUG_ENABLED? Or is it possible to work on worker/job worker as well?

Reply:
it works on all grpc servers

Reply:
i'll change the name